mbot_nlu_training/common/src/mbot_nlu_training/eegpsr/slots_NN/eegpsr_data_generator.py

```diff
@@ -186,15 +191,15 @@
 intros = ['robot', 'hello robot', 'hello', 'please', 'could you please', 'robot please', 'can you', 'robot can you', 'robot could you', 'could you']
 
 # Prining number of objects used in the generator
-print('obect_a', len(objects_a))
-print('obect_an', len(objects_an))
-print('objects_the', len(objects_the))
-print('objects_some', len(objects_some))
-print('objects_a_piece_of', len(objects_a_piece_of))
-print('objects_a_cup_of', len(objects_a_cup_of))
-print('objects_a_can_of', len(objects_a_can_of))
-print('objects_a_glass_of', len(objects_a_glass_of))
-print('objects_a_bottle_of', len(objects_a_bottle_of))
+# print('obect_a', len(objects_a))
```

Comment:
commented out lines? either remove or justify why are we leaving them please.

Reply:
Cleared.